iocore/aio/AIO.cc

```diff
@@ -442,8 +440,10 @@ AIOThreadInfo::aio_thread_main(AIOThreadInfo *thr_info)
 #ifdef AIO_STATS
```

Comment:
How the outer `for` loop not an infinite loop?  The `break` on line 438 breaks out of the `do` loop only, no?